tl/console-and-shells/i18n-shell.rst

```diff
@@ -0,0 +1,113 @@
+I18N Shell
+##########
+
+<<<<<<< HEAD
+The i18n features of CakePHP use `po files <http://en.wikipedia.org/wiki/GNU_gettext>`_
+as their translation source. PO files integrate with commonly used translation tools
+like `Poedit <http://www.poedit.net/>`_.
+=======
```

Comment:
You have merge conflict markers her ad in a few other files. Those will need to be cleaned up.